packages/trader/src/Modules/Trading/Containers/trade.jsx

```diff
@@ -130,6 +132,8 @@ const Trade = observer(() => {
         setSwipeIndex(index);
     };
 
+    const is_already_shown = JSON.parse(sessionStorage.getItem('launchModalShown') ?? 'false');
```

Comment:
We gonna show the modal every new modal?

Reply:
Seems that it'll be shown even if your open another tab with DTrader

Reply:
It is saved in localStorage now. We'll not show until the user logouts and log in again .